src/dotnetsheff.Api/FeedbackModule.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Autofac;
+using dotnetsheff.Api.GetAvailableFeedbackEvents;
+using dotnetsheff.Api.Meetup;
+
+namespace dotnetsheff.Api
+{
+    public class FeedbackModule : Module
+    {
+        private const string GROUP_NAME = "dotnetsheff";
+        protected override void Load(ContainerBuilder builder)
+        {
+            builder
```

Comment:
what do you think @kevbite ?

Reply:
Scanning assembly for types? seems fine to me 👍